consumer.go

```diff
@@ -39,6 +39,15 @@ func (ce ConsumerErrors) Error() string {
 // on a consumer to avoid leaks, it will not be garbage-collected automatically when it passes out of
 // scope.
 type Consumer interface {
+
+	// Topics returns the set of available topics as retrieved from the cluster metadata.
+	// This methods delegates to the internal Client instance.
+	Topics() ([]string, error)
+
+	// Partitions returns the sorted list of all partition IDs for the given topic.
+	// This methods delegates to the internal Client instance.
+	Partitions(topic string) ([]int32, error)
```

Comment:
Rather than mention delegation (technical aspect) mention that they are "the same as" and "provided for convenience" or something (UX aspect).